collaboration.py

```diff
@@ -89,7 +89,7 @@ def greet_horia_tudor_tatu():
 	return 'missing'
 
 def greet_andra_vlad():
-	return 'missing'
+	return 'Andra-Vlad'
```

Comment:
I think you should exclude the `-`

Reply:
[f573e73](https://github.com/pvl26/hackademy-pycolab/pull/112/commits/f573e734e612c44eb3f7430e4a227a1ed081ebfa)